code/extensions/FluentFilteredExtension.php

```diff
@@ -9,13 +9,6 @@
 class FluentFilteredExtension extends DataExtension
 {
     /**
-     * Data query key necessary to turn on admin filtering
-     *
-     * @var string
-     */
```

Comment:
I just realised something... if we remove this, it breaks backwards compatibility, since existing code could refer to it.

Can you please leave this block in? You don't have to use it, so the below code is ok.